### PR TITLE
HPCC-15464 Query priority in Roxie being ignored

### PR DESCRIPTION
**Pull request by richardkchapman**
Signed-off-by: Richard Chapman rchapman@hpccsystems.com

### CODE REVIEW

**Comment by richardkchapman** [comment]
@stuartort @ghalliday Please review

File: roxie/ccd/ccdquery.cpp [hunk 1]
```diff
@@ -515,6 +524,9 @@ class CQueryFactory : public CInterface, implements IQueryFactory, implements IR
         if (!helperFactory)
             throw MakeStringException(ROXIE_INTERNAL_ERROR, "Internal error: helper function %s not exported", helperName.str());
 
+        RemoteActivityId remoteId(rid, hashValue);
+        RemoteActivityId remoteId2(rid | ROXIE_ACTIVITY_FETCH, hashValue);
+
```
Comment (stuartort) [inline roxie/ccd/ccdquery.cpp]: remoteId id used in most of the case statements below so it makes sense to make the call all the time, but remoteId2 is only use in the 1 case for TAKkeyedjoin, TAKkeyeddenormalize, TAKkeyeddenormalizegroup - should we always create it?

Comment (ghalliday) [inline roxie/ccd/ccdquery.cpp]: It could do.  The overhead is tiny, and the code is not time critical, so I would take it as it stands.

**Comment by stuartort** [comment]
Based on Gavin's comment, this looks fine

**Pull request merged**